eng/Versions.props

```diff
@@ -68,7 +68,7 @@
     <MicrosoftCodeAnalysisVisualBasicCodeFixTestingXUnitVersion>$(MicrosoftCodeAnalysisTestingVersion)</MicrosoftCodeAnalysisVisualBasicCodeFixTestingXUnitVersion>
     <MicrosoftCodeAnalysisVisualBasicCodeRefactoringTestingXUnitVersion>$(MicrosoftCodeAnalysisTestingVersion)</MicrosoftCodeAnalysisVisualBasicCodeRefactoringTestingXUnitVersion>
     <MicrosoftCodeAnalysisVisualBasicCodeStyleVersion>$(CodeStyleAnalyzerVersion)</MicrosoftCodeAnalysisVisualBasicCodeStyleVersion>
-    <MicrosoftCodeAnalysisAnalyzerUtilitiesVersion>3.0.0</MicrosoftCodeAnalysisAnalyzerUtilitiesVersion>
+    <MicrosoftCodeAnalysisAnalyzerUtilitiesVersion>3.3.0</MicrosoftCodeAnalysisAnalyzerUtilitiesVersion>
```

Comment:
FYI: This would need a corresponding version change in the VS repo [here](https://devdiv.visualstudio.com/DevDiv/_git/VS?path=%2F.corext%2FConfigs%2Fdefault.config&version=GBmaster&line=114&lineEnd=114&lineStartColumn=5&lineEndColumn=157&lineStyle=plain) once we insert Roslyn with this change.

Reply:
FYI @RikkiGibson